### ISSUE TITLE
"Non-Vaal Skills deal X% increased Damage during Soul Gain Prevention" combat option?

### ISSUE THREAD

**Issue by Protuhj** [PathOfBuildingCommunity/PathOfBuilding#1672]
Trying to view the damage difference for the "Non-Vaal Skills deal (51-60)% increased Damage during Soul Gain Prevention" crafted modifier, but the option to toggle soul gain prevention on or off does not appear in the "When in Combat" options section.

Am I missing something?

```
Rapture Road
Samite Slippers
Unique ID: f91b3c2cb978c9cd66628a00326c5b348569fda492d42d04d595f840aaea1be1
Item Level: 72
Quality: 20
Sockets: B-B-G-B
LevelReq: 44
Implicits: 2
{crafted}8% chance to Dodge Spell Hits if you've
{crafted}taken Spell Damage Recently
+49 to maximum Life
+30 to maximum Mana
+19% to Cold Resistance
+30% to Lightning Resistance
30% increased Movement Speed
{crafted}Non-Vaal Skills deal (51-60)% increased Damage during Soul Gain Prevention
```

**Comment by Protuhj** [comment]
The existing form ["^(%d+)%% increased"] = "INC" isn't broad enough, especially considering it has the "line start ^" qualifier at the beginning of the pattern, which excludes this specific mod form.

**Comment by Protuhj** [comment]
Adding the preFlagList entry allowed the soul gain prevention box to show, and doesn't increase Vaal skill damage like just adding the modForm did.

**Comment by ppoelzl** [comment]
Fixed in 1.4.170.19.
